man/systemd-vmspawn.xml

```diff
@@ -413,7 +413,7 @@
 
           <listitem><para>Controls user namespacing under <option>--directory=</option>.
           If enabled,
-          <citerefentry><refentrytitle>virtiofsd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+          <citerefentry project='url'><refentrytitle url='https://manpages.debian.org/bookworm/qemu-system-common/virtiofsd.1.en.html'>virtiofsd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
```

Comment:
maybe point this to unstable `https://manpages.debian.org/unstable/qemu-system-common/virtiofsd.1.en.html` so that there's a chance it will be more up to date, like links to man7 and others?

Reply:
Ah, I actually edited the url to use `sid`, but then I pasted the wrong thing into the man page.